assets/scss/objects/_container.scss

```diff
@@ -0,0 +1,11 @@
+.o-container {
+    padding: 1.5rem;
```

Comment:
When you create a container object, it's good to set `width` and `max-width` so the content won't get stretched to the viewport. But you don't need to do it in this case, as I'll merge my changes to the same file 😅